tests/manage/monitoring/conftest.py

```diff
@@ -341,10 +341,12 @@ def teardown():
     logger.info(f"Put object into {pool_name}")
     pool_object = "test_object"
     ct_pod.exec_ceph_cmd(f"rados -p {pool_name} put {pool_object} /etc/passwd")
+    logger.info(f"Corrupting pool {pool_name} on {osd_deployment.name}")
+    rados_utils.corrupt_pg(osd_deployment, pool_name, pool_object)
 
     def corrupt_pg():
```

Comment:
to leave the internal function corrupt_pg() which will now sleep and return osd_deployment.name will be misleading as we use it as an 'opetation' for measure_operation call. May be rename it? or simplify and remove it? WDYT?

Reply:
Good point, name of the function and docstring were changed.